File: java/src/main/java/com/a65apps/kalimullinilnazrafilovich/interactors/details/ContactDetailsInteractor.java
package com.a65apps.kalimullinilnazrafilovich.interactors.details;

import com.a65apps.kalimullinilnazrafilovich.entities.ContactDetailsInfo;

import io.reactivex.rxjava3.core.Single;

public interface ContactDetailsInteractor {
    Single<ContactDetailsInfo> loadDetailsContact(String id);
}

File: java/src/main/java/com/a65apps/kalimullinilnazrafilovich/interactors/details/ContactDetailsInteractor.kt
package com.a65apps.kalimullinilnazrafilovich.interactors.details

import com.a65apps.kalimullinilnazrafilovich.entities.ContactDetailsInfo
import kotlinx.coroutines.flow.Flow

interface ContactDetailsInteractor {
    fun loadDetailsContact(id: String): Flow<ContactDetailsInfo>
}
